Description: Add simple manpage for flnet binary
 .
 flnet (7.2.3-1) unstable; urgency=medium
 .
   * New upstream release v7.2.3
   * Initial Debian Release (Closes: #774925)
Author: Greg Beam <ki7mt@yahoo.com>
Bug-Debian: http://bugs.debian.org/774925

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ flnet-7.2.3/manpages/flnet.1
@@ -0,0 +1,74 @@
+'\" t
+.\"     Title: flnet
+.\"    Author: [see the "AUTHORS" section]
+.\" Generator: DocBook XSL Stylesheets v1.78.1 <http://docbook.sf.net/>
+.\"      Date: 08/12/2015
+.\"    Manual: FLNET Man Page
+.\"    Source: FLNET Version 7.2.3
+.\"  Language: English
+.\"
+.TH "FLNET" "1" "08/12/2015" "FLNET Version 7\&.2\&.3" "FLNET Man Page"
+.\" -----------------------------------------------------------------
+.\" * Define some portability stuff
+.\" -----------------------------------------------------------------
+.\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.\" http://bugs.debian.org/507673
+.\" http://lists.gnu.org/archive/html/groff/2009-02/msg00013.html
+.\" ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.ie \n(.g .ds Aq \(aq
+.el       .ds Aq '
+.\" -----------------------------------------------------------------
+.\" * set default formatting
+.\" -----------------------------------------------------------------
+.\" disable hyphenation
+.nh
+.\" disable justification (adjust text to left margin only)
+.ad l
+.\" -----------------------------------------------------------------
+.\" * MAIN CONTENT STARTS HERE *
+.\" -----------------------------------------------------------------
+.SH "NAME"
+flnet \- amateur radio logbook server
+.SH "SYNOPSIS"
+.sp
+\fBflnet\fR \- provides the Amateur Radio Net Control Station operator with a real time tool to assist him or her in managing the net activities\&. A single screen with multiple windows is used to allow rapid entry, search, pick and display of all stations calling in to the net\&. All operations on the main screen are accomplished with keyboard entries only\&. No mouse action is required to perform the net control functions\&. Experience has shown that most net control operators prefer this method of operation to improve the speed of entry and selection\&.
+.SH "BUGS"
+.sp
+If you find a bug or suspect \fB\fIflnet\fR\fR is not acting as you think it should, send an email with as much detail as possible to: <fldigi\-devel@lists\&.sourceforge\&.net>
+.SH "AUTHORS"
+.sp
+.if n \{\
+.RS 4
+.\}
+.nf
+Dave Freeze, W1HKJ, <w1hkj@w1hkj\&.com>
+.fi
+.if n \{\
+.RE
+.\}
+.SH "RESOURCES"
+.sp
+.if n \{\
+.RS 4
+.\}
+.nf
+Project Site: \&.\&.\&. <http://sourceforge\&.net/projects/fldigi/>
+Main web site: \&.\&. <http://www\&.w1hkj\&.com/Fldigi\&.html>
+Documentation: \&.\&. <http://www\&.w1hkj\&.com/fllog\-help/index\&.html>
+.fi
+.if n \{\
+.RE
+.\}
+.SH "DEBIAN MAINTAINERS"
+.sp
+.if n \{\
+.RS 4
+.\}
+.nf
+Maintainer \&.\&.: Debian Hamradio Maintainers <debian\-hams@lists\&.debian\&.org>
+Uploaders \&.\&.\&.: Greg Beam <ki7mt@yahoo\&.com>, Kamal Mostafa <kamal@whence\&.com>
+IRC \&.\&.\&.\&.\&.\&.\&.\&.\&.: OFTC #debian\-hams, FREENODE: #ubuntu\-hams
+.fi
+.if n \{\
+.RE
+.\}
--- /dev/null
+++ flnet-7.2.3/manpages/flnet.1.txt
@@ -0,0 +1,53 @@
+// Process this file with AsciiDoc:
+// Command: a2x.py --doctype manpage --format manpage --no-xmllint flnet.1.txt
+:doctype: manpage
+:man source: FLNET
+:man manual: FLNET Man Page
+:man date: {date}
+:man version: Version 7.2.3
+
+= flnet(1)
+
+== NAME
+
+flnet - amateur radio logbook server
+
+
+== SYNOPSIS
+
+*flnet* - provides the Amateur Radio Net Control Station operator with a real
+time tool to assist him or her in managing the net activities.  A single
+screen with multiple windows is used to allow rapid entry, search, pick
+and display of all stations calling in to the net.  All operations on
+the main screen are accomplished with keyboard entries only.  No mouse
+action is required to perform the net control functions.  Experience
+has shown that most net control operators prefer this method of operation
+to improve the speed of entry and selection.
+
+
+== BUGS
+
+If you find a bug or suspect *'flnet'* is not acting as you think it should,
+send an email with as much detail as possible to: <fldigi-devel@lists.sourceforge.net>
+
+
+== AUTHORS
+-----
+Dave Freeze, W1HKJ, <w1hkj@w1hkj.com>
+-----
+
+[[X1]]
+== RESOURCES
+-----
+Project Site: ... <http://sourceforge.net/projects/fldigi/>
+Main web site: .. <http://www.w1hkj.com/Fldigi.html>
+Documentation: .. <http://www.w1hkj.com/fllog-help/index.html>
+-----
+
+== DEBIAN MAINTAINERS
+-----
+Maintainer ..: Debian Hamradio Maintainers <debian-hams@lists.debian.org>
+Uploaders ...: Greg Beam <ki7mt@yahoo.com>, Kamal Mostafa <kamal@whence.com>
+IRC .........: OFTC #debian-hams, FREENODE: #ubuntu-hams
+-----
+
